examples/consume/app.ts

```diff
@@ -9,7 +9,14 @@ const consume = async (consumer: Consumer) => {
   consumer.on(EventType.Response, async (message: ResponseMessage) => {
     const { offset, topic, value } = message.content()
 
-    console.log('Message received !', { envelopeId: value.id, offset, topic, chain: value.chain })
+    console.log('Message received !', {
+      Id: value.id,
+      jobId: value.jobId,
```

Comment:
I think we should stick with JobUUID (UUID instead of ID) everywhere.

Reply:
Agree, renaming